test/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/address/CompanyDetailsControllerSpec.scala

```diff
@@ -140,6 +141,39 @@ class CompanyDetailsControllerSpec
     (sessionData, fillingOutReturn, draftReturn)
   }
 
+  def sessionWithDraftMultipleIndirectDisposals(
+    individualUserType: IndividualUserType
+  ): (SessionData, FillingOutReturn, DraftMultipleIndirectDisposalsReturn) = {
+    val companyAddress   = sample[Address]
+    val country          = Country("HK", Some("Hong Kong"))
+    val draftReturn      = sample[DraftMultipleIndirectDisposalsReturn].copy(
+      triageAnswers = sample[CompleteMultipleDisposalsTriageAnswers].copy(
+        countryOfResidence = country,
+        assetTypes = List(AssetType.IndirectDisposal),
+        individualUserType = Some(individualUserType)
+      ),
+      exampleCompanyDetailsAnswers = Some(
+        sample[CompleteExampleCompanyDetailsAnswers].copy(
+          address = companyAddress
+        )
+      )
+    )
+    val fillingOutReturn = sample[FillingOutReturn].copy(
+      draftReturn = draftReturn
+    )
+    val sessionData      = SessionData.empty.copy(
+      journeyStatus = Some(fillingOutReturn)
+    )
+
+    (sessionData, fillingOutReturn, draftReturn)
+  }
+
+  val acceptedIndividualUserTypeGen: Gen[IndividualUserType] =
+    individualUserTypeGen.filter {
+      case Self | Capacitor | PersonalRepresentative => true
```

Comment:
we should be able to update this to include `PeronslaRepresentativeInPeriodOfAdmin` now. But then that's all `IndividualUserType`, so we might not need `acceptedIndividualUserTypeGen` at all now

Reply:
As this spec is not organised  properly to handle different User types and IndividualUserTypes, I have tested Non-POA using acceptedIndividualUserTypeGen and PeronslaRepresentativeInPeriodOfAdmin in separate tests.

Will try to merge them into one.